enumerable.rb

```diff
@@ -0,0 +1,31 @@
+module MyEnumerable
+  def all?
+    each { |element| return false unless yield(element) }
+    true
+  end
+
+  def any?
+    each { |element| return true if yield(element) }
+    false
+  end
+
+  def filter
+    result = []
+    each { |element| result << element if yield(element) }
+    result
+  end
+end
+
+class MyList
+  include MyEnumerable
+  def initialize(*list)
```

Comment:
Great job @obed300 🤝

- [ ] Kindly ensure the `MyClass` and `MyEnumerable` are saved in two separate files.
- [ ] Kindly ensure that the student uses `require_relative` to import `MyEnumerable` since using `require_relative` for files in your directory instead of `require` is a ruby convention.